.travis.yml

```diff
@@ -3,12 +3,13 @@ language: ruby
 sudo: false
 
 rvm:
+  - 2.3.1
   - 2.3.0
-  - 2.2
-  - 2.1
-  - 2.0.0
+  - 2.2.5
```

Comment:
I think '2.2' will pick up 2.2.0, not 2.2.5 (which is the latest 2.2.x release).  How about 2.2.5 and 2.3.1?  (omitting 2.2.2, 2.2.3, 2.2.4, and 2.3.0)